src/app/cold-storage/rebalance-to-cold-wallet.ts

```diff
@@ -58,16 +59,16 @@ export const rebalanceToColdWallet = async (): Promise<boolean | ApplicationErro
 
   const description = `deposit of ${rebalanceAmount} sats to the cold storage wallet`
 
-  const usd = rebalanceAmount * usdPerSat
-  const usdFee = fee * usdPerSat
+  const amountDisplayCurrency = DisplayCurrencyConversionRate(usdPerSat)(rebalanceAmount)
+  const feeDisplayCurrency = DisplayCurrencyConversionRate(usdPerSat)(fee)
```

Comment:
Similar point from before, it looks like this is always called as `DisplayCurrencyConversionRate(usdPerSat)(amount)`. Currying may be redundant here.

Reply:
not using the currying now, but could be useful in the future. we can also see the first parameter is the `Input` and the second is the `Args`

Reply:
Yup, I'm curious how different this pattern is from things like our `WalletInvoiceValidator` that takes an invoice in the constructor and then has method like `validateToSend` that takes args

Reply:
I'm with @vindard - don't see a need for currying here. Wouldn't introduce the pattern until its really necessary. Would just stick with the constructor `XxxConfig` and method `xxxArgs` type patterns.